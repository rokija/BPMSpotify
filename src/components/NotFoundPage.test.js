import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFoundPage from './NotFoundPage';

describe('NotFoundPage', () => {
    const wrapper = shallow(<NotFoundPage />);

    it('renders properly', () => {
        expect(wrapper.length).toEqual(1);
    });

    it('matches snapshot', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
