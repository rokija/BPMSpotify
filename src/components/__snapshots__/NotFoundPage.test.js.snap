// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage matches snapshot 1`] = `
<div
  className="not-found-page"
>
  <h4>
    404 Page Not Found
  </h4>
  <Link
    replace={false}
    to="/"
  >
     Go back to homepage 
  </Link>
</div>
`;
